addons/vultr/helpers/ProductsHelper.php

```diff
@@ -116,7 +116,7 @@ public static function getProductConfigOptions($productID, $field = 'all', $defa
 	public static function customFields($productID)
 	{
 		$result = DB::table('tblcustomfields')->where('fieldname', 'subid|Virtual machine ID')->where('type', 'product')->where('relid', $productID)->select('id')->get();
-		if (!$result)
+		if (count($result) < 1)

```

Comment:
This appears to be a good and necessary change needed for php72+ from what i can tell